examples/navier-stokes/densitycurrent.h

```diff
@@ -77,7 +77,8 @@
 //   center          ,  Location of bubble center
 //   dc_axis         ,  Axis of density current cylindrical anomaly, or {0,0,0} for spherically symmetric
 // *****************************************************************************
-static int Exact_DC(CeedInt dim, CeedScalar time, const CeedScalar X[], CeedInt Nf, CeedScalar q[], void *ctx) {
+static int Exact_DC(CeedInt dim, CeedScalar time, const CeedScalar X[],
+                    CeedInt Nf, CeedScalar q[], void *ctx) {
```

Comment:
`make style` had a bunch of changes for `densitycurrent.h`, and I rejected about half of them. I need to look into how to get astyle to stop trying to add strange spacing for some of the multiplication in this file.

Unrelated - we may wish to consider putting all the QFunction headers in a separate folder. We did this for the petsc folder and it seems to make it easier for new users to navigate.